app/models/participant_profile.rb

```diff
@@ -3,7 +3,7 @@
 class ParticipantProfile < ApplicationRecord
   belongs_to :user
   belongs_to :school
-  belongs_to :cohort
+  belongs_to :cohort, optional: true
```

Comment:
Do we really need to relax this? I know mentor profiles technically have it as optional, but I don't think any live ones should have that problem :P

Reply:
Yeah, but ones created in tests do...

Reply:
I think we did something to around making it non-optional for ects recently, and I think refactoring tests was a part of that (https://github.com/DFE-Digital/early-careers-framework/pull/562).

Do you want to see if we can use the same idea for a quick fix for mentor profiles in tests?

Reply:
Well, most likely this is just a matter of adding `cohort` to the factory. If we are certain that this should not be optional I'm happy to change this :)

Reply:
I am certain this should not be optional. Mentors without cohorts should not exist with our current setup, and I don't think we know of any cases when they should - it was just that mentor profile was copy-pasted from ECT profile at a time when we haven't fixed the same issue on ECT profiles.